he.json

```diff
@@ -0,0 +1,297 @@
+                "th-date" : "תאריך",
+                "th-days" : "מספר ימים לפני"
+            },
+            "set" : "סט",
+            "set-name" : "חלק מ[name]",
+            "popular-title" : "אייטמים פופולאריים היום"
+        },
+        "account" : {
+            "home" : {
+                "title" : "נתוני החשבון שלך",
+                "error" : "משהו השתבש בזמן טעינת נתוני החשבון שלך, אנא נסה שוב מאוחר יותר.",
+                "info" : {
+                    "title" : "מידע",
+                    "line1" : "אתה מחובר בתור [username] בעזרת [provider]",
+                    "line2-api-yes" : "יש לך[גישה לapi](/api/docs)",
+                    "line2-api-no" : "אין לך גישה לapi."
+                },
+                "migration" : {
+                    "title" : "חשבונות קודמים",
+                    "waiting-line1" : "עברנו להשתמש בדיסקורד לצורך התחברות.",
+                    "waiting-line2" : "אם יש לך חשבון עוד מ2018, תצטרך להתחבר אליו ולשייך אותו לחשבון הזה",
+                    "twitter-line1" : "עברנו להשתמש בדיסקורד לצורך התחברות. אתה כרגע משתמש בחשבון מדור קודם שעלול להיפסק בעתיד ושכבר עכשיו בעל פונקציונליות מוגבלת.",
+                    "twitter-line2" : "כדי לעבור לחשבון דיסקורד, לחץ על הכפתור מטה"
+                },
+                "ratings" : {
+                    "title" : "דירוגים",
+                    "none" : "אין לך דירוגים."
+                },
+                "destroy" : {
+                    "title" : "השמד חשבון",
+                    "line1" : "אם אינך רוצה עוד את חשבונך, תוכל למחוק אותו ואת כל הנתונים שלו [כאן](/account/destroy)"
+                }
+            },
+            "oauth" : {
+                "title" : "התחבר לfnbr.co",
+                "discord-button" : "התחבר בעזרת דיסקורד",
+                "legacy-text" : "לחלופין, אם יש לך חשבון דור קודם משנת 2018 הזקוק להעברה:",
+                "twitter-button" : "התחבר בעזרת טוויטר",
+                "privacy-text" : "על ידי כניסה אתה מסכים לנו לאחסן נתונים המשויכים לחשבונך כמפורט [במדיניות הפרטיות](/privacy) שלנו."
+            },
+            "username" : {
+                "title" : "ברוך הבא לfnbr.co",
+                "line1" : "אתה מחובר בעזרת דיסקורד. כדי ליצור אינטרקציה עם אחרים באתר, תצטרך לבחור שם משתמש",
+                "nameTaken" : "שם המשתמש שלך [attemptedName] נלקח על ידי משתמש אחר, ולכן הוספנו לו את מאבחן הדיסקורד שלך ואנו ממליצים לך להשתמש בשם זה.",
+                "randomName" : "שם המשתמש שלך [attemptedName] נלקח על ידי משתמש אחר, ולא הצלחנו ליצור לך שם משתמש יחודי בעזרת מאבחן משתמש הדיסקורד שלך.",
+                "form-username" : "שם משתמש",
+                "form-criteria" : "אנא בחר שם משתמש בין 2 ל-32 תווים, אשר יכיל אותיות באנגלית,  קווים תחתונים ומקפים בלבד.",
+                "button-text" : "קבע את שם המשתמש שלי"
+            },
+            "locked" : {
+                "title" : "ההתחברות נכשלה",
+                "line1" : "אינך יכול להתחבר לfnbr.co מכיוון [שחשבונך ננעל](class=text-danger)"
```

Comment:
It's not allowed to separate the text (within the square brackets `[...]`) from the styling option (inside the round brackets `(...)`). It must be in this format: `[text](class=text-danger)`